website/docs/d/cloudtrail_service_account.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "aws"
+page_title: "AWS: aws_cloudtrail_service_account"
+sidebar_current: "docs-aws-datasource-cloudtrail-service-account"
+description: |-
+  Get AWS CloudTrail Service Account ID for storing trail data in S3.
+---
+
+# aws\_cloudtrail\_service\_account
```

Comment:
These antislashes are not needed anymore when writing markdown

Reply:
OK, I'll correct.